runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -122,6 +122,9 @@
 	@Inject
 	private ClanManager clanManager;
 
+	@Inject
+	private ClientLoader clientLoader;
```

Comment:
Theres really no reason to keep this around/it has no dependencies in it... its probably not worth moving?